desktop/core/src/desktop/management/commands/sync_warehouses.py

```diff
@@ -0,0 +1,203 @@
+#!/usr/bin/env python
```

Comment:
how frequently does the sync warehouse happen? Is this script responsible for deleting VW which has gone away?